HHS/OPRE-OPS
# OPS-2193/task-2238 BLI Change Requests Agreement History

## What changed

* Agreement History 
  * BLI change requests are now including as IN_REVIEW changes for the BLI
  * API now includes `log_items` which contains changes broken down by property like it has been in the Agreement History panel in the FE
  * FE Agreement History panel reworked use `log_items` from the BE instead of doing all that in the FE
* bug fix for `getNotificationsByUserId` to not query for notifications if there's not a user ID (OIDC)
* `tests_manual` folder added for running test manual against an already running app (docker)

## Issue

#2193 #2238 

continues PRs: #2232 #2205 

## How to test

* run the tests
* edit an Agreement and a DRAFT BLI and verify the Agreement history is still working as before
* edit a BLI that is PLANNED and verify that the change requests for these are in the Agreement history

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/127965976/cbbb08cb-9ad2-4170-b2da-9d5a0aaf0f79)


## Definition of Done Checklist
